src/helpers/handleIncrementalUpdate.ts

```diff
@@ -42,5 +42,5 @@ export const handleIncrementalUpdateArray = (depthOld: string[][], newLevels: st
         return sortAsks(result);
     }
 
-    return sortBids(result);
+    return sortBids(result.filter(order => order[1])); //order[0] - price, order[1] - amount
```

Comment:
Looks wrong to me. You did all operations with a data and after all filtering.
You need to check if second elements of ```newLevels``` exists in the start of the function

Reply:
And we can't send depthOld, as at the start of the function, because it can return orders, which were canceled or matched